index.html

```diff
@@ -37,7 +37,61 @@ <h1>
         </div>
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure your design looks similar to the original template. as you may notice how different they are. sticking to the original design will give your page a great look and feel for a better user experience.
- [ ] Make sure your hamburger menu is placed on the right as is the standard for UX and better user experience on the mobile screen and also make sure the background image is well set as is the case in the original design.

### Your design :point_down: 
![image](https://github.com/Arch-Noize/setup-and-mobile-first/assets/97100091/3164c95e-dd97-4241-9ef2-a914fc71524a)

### Original template :point_down: 
![image](https://github.com/Arch-Noize/setup-and-mobile-first/assets/97100091/7ab264b2-ce67-40cc-b12d-572aee08e7a1)